blocks/article-grid/article-grid.js

```diff
@@ -0,0 +1,364 @@
+import { getLibs, buildBlock } from '../../scripts/utils.js';
+import initBanner from '../banner/banner.js';
+
+const miloLibs = getLibs();
+const defaultLimit = 9;
+const ROOT_MARGIN = 20;
+
+// Used for getting localized text
+const replacePlaceholder = async (key) => {
+  const { replaceKey } = await import(`${miloLibs}/features/placeholders.js`);
+  const { getConfig } = await import(`${miloLibs}/utils/utils.js`);
+  const result = await replaceKey(key, getConfig());
+
+  return result;
+};
```

Comment:
Load the two files in parallel

```suggestion
const replacePlaceholder = async (key) => {
  const [{ replaceKey }, { getConfig }] = await Promise.all([
    import(`${miloLibs}/features/placeholders.js`),
    import(`${miloLibs}/utils/utils.js`)
  ]);

  const result = await replaceKey(key, getConfig());

  return result;
};
```

Reply:
Thanks! Updated and moved the function to `scripts/utils.js` as `replacePlaceholderForLocalizedText` for reusability